docs/resources/observability_pipeline.md

```diff
@@ -3,12 +3,16 @@
 page_title: "datadog_observability_pipeline Resource - terraform-provider-datadog"
 subcategory: ""
 description: |-
-  Provides a Datadog Observability Pipeline resource. Observability Pipelines allows you to collect and process logs within your own infrastructure, and then route them to downstream integrations.
+  Provides a Datadog Observability Pipeline resource. Observability Pipelines allows you to collect and process logs within your own infrastructure, and then route them to downstream integrations.This resource is in Preview. Please reach out to Datadog support to enable it for your account.
+  We recommend using -parallelism=1 option to apply this resource.
```

Comment:
Suggesting:
> Datadog recommends using the `-parallelism=1` option to apply this resource.

Reply:
Didn't realize I couldn't add a suggestion because I was viewing a specific commit 🤦‍♀️. I added the suggestions below.